.github/workflows/test.yaml

```diff
@@ -1,25 +1,33 @@
-name: test helm charts
-on:
-  workflow_call:
-
+name: Install Helm Charts
+on: [workflow_call]
 jobs:
   install:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: [kube-1-23-9, kube-1-24-2, kube-1-25-4]
+        version: [kube-1-23-9, kube-1-24-2, kube-1-25-4, kube-1-26-0]
         capi: ["v1.2.5"]
-        capo: ["v0.7.0-stackhpc.1"]
+        capo: ["v0.7.0-stackhpc.3"]
         addonprovider: ["0.1.0-dev.0.main.21"]
-    continue-on-error: true # don't cancel all jobs if one fails - cleanup
+      max-parallel: 1
+    continue-on-error: true #don't cancel all jobs if one fails - cleanup
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
       
       - name: Install tools
         run: sudo apt install -y zip unzip 
       
+      - name: Check if sonobuoy is installed
+        id: sonobuoy_exists
+        uses: andstor/file-existence-action@v2
```

Comment:
Does this need an action? Could just use bash here and condense the Install Sonobouy step into one:

```
if [ ! -x /usr/bin/sonobouy ]; then
    curl -fsSL https://github.com/vmware-tanzu/sonobuoy/releases/download/v${SONOBOUY_VERSION}/sonobuoy_${SONOBOUY_VERSION}_linux_amd64.tar.gz | tar -xz
fi
```

Could also parameterise Sonobouy version